### PR TITLE
Adding catch for blocked autoplay

### PR DESCRIPTION
**Pull request by bcfancher-mc**
## Overview
there was an issue discovered where autoplay was throwing an exception.  This is as designed per browser autoplay policies

image

Google AutoPlay Policy Changes:
https://developers.google.com/web/updates/2017/09/autoplay-policy-changes#new-behaviors

## Risks
None

## Changes
Before:
image

After:
image

## Issue
N/A

## Breaking change?
Backwards Compatible

### CODE REVIEW

File: src/components/VideoPlayer/index.js [hunk 1]
```diff
@@ -165,7 +165,10 @@ export default class VideoPlayer extends PureComponent {
     }
 
     if (hasAutoplay) {
-      this.video.play()
+      const promise = this.video.play()
+      if (promise !== undefined) {
```
Comment (a-axton) [inline src/components/VideoPlayer/index.js]: why is this part needed? is it undefined under certain conditions?

Comment (bcfancher-mc) [reply src/components/VideoPlayer/index.js]: not sure, it's what was recommended on the google docs.

Comment (bcfancher-mc) [reply src/components/VideoPlayer/index.js]: I'd rather play it safe-than-sorry, given this is currently plaguing the packages player

Comment (jstnjns) [reply src/components/VideoPlayer/index.js]: word.  we can improve later if we find we want to
